File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1061.py
di = int(input().split()[1])
ti = input()
df = int(input().split()[1])
tf = input()

hi = int(ti[0:2])
mi = int(ti[5:7])
si = int(ti[10:12])

hf = int(tf[0:2])
mf = int(tf[5:7])
sf = int(tf[10:12])

st=(sf-si)+(mf*60-mi*60)+(hf*3600-hi*3600)+(86400*df-86400*di);

print("{} dia(s)\n{} hora(s)".format(st//86400, (st%86400)//3600))
print("{} minuto(s)\n{} segundo(s)".format(((st%86400)%3600)//60, (((st%86400)%3600)%60)))

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1061.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    int i, di, hi, mi, si, df, hf, mf, sf, st;
    string s, ditxt="";
    getline(cin, s);
    for(i=4;i<s.size();i++){
        ditxt+=s[i];
    }
    di=stoi(ditxt);
    
    getline(cin, s);
    ditxt="";
    ditxt+=s[0];
    ditxt+=s[1];
    hi=stoi(ditxt);
    
    ditxt="";
    ditxt+=s[5];
    ditxt+=s[6];
    mi=stoi(ditxt);

    ditxt="";
    ditxt+=s[10];
    ditxt+=s[11];
    si=stoi(ditxt);
    
    getline(cin, s);
    ditxt="";
    for(i=4;i<s.size();i++){
        ditxt+=s[i];
    }
    df=stoi(ditxt);

    getline(cin, s);
    ditxt="";
    ditxt+=s[0];
    ditxt+=s[1];
    hf=stoi(ditxt);
    
    ditxt="";
    ditxt+=s[5];
    ditxt+=s[6];
    mf=stoi(ditxt);

    ditxt="";
    ditxt+=s[10];
    ditxt+=s[11];
    sf=stoi(ditxt);

    st=(sf-si)+(mf*60-mi*60)+(hf*3600-hi*3600)+(86400*df-86400*di);
    
    cout << st/86400 << " dia(s)" << endl << (st%86400)/3600 << " hora(s)" << endl;
    cout << ((st%86400)%3600)/60 << " minuto(s)" << endl << (((st%86400)%3600)%60) << " segundo(s)" << endl;
        
    return 0;
}